### PR TITLE
[docs] Make the link to initial setup stand out.

### PR DESCRIPTION
**Pull request by colinschmidt**
As discussed no one reads anything so we make this very important notice look very important.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -10,10 +10,8 @@ Welcome to Chipyard's documentation!
 
 Chipyard is a framework for designing and evaluating full-system hardware using agile teams.
 It is composed of a collection of tools and libraries designed to provide an integration between open-source and commercial tools for the development of systems-on-chip.
-New to Chipyard? Jump to the :ref:`Chipyard Basics` page for more info.
 
-Quick Start
-===============================
+.. IMPORTANT:: **New to Chipyard?** Jump to the :ref:`Chipyard Basics` page for more info.
 
 System Requirements
```
Comment (sagark) [inline docs/index.rst:16]: Don't we want to remove this stuff too?

**Comment by sagark** [comment]
I thought the agreement was to get rid of the duplicated installation instructions on the index page?

File: docs/index.rst (hunk 1 above)
Comment (colinschmidt) [reply docs/index.rst:16]: Yes we do but I wanted someone who knew if anything needed to be kept to make the deletion.
I'm not sure they are 100% redundant.

**Comment by sagark** [comment]
Also, depending on when we're planning to ship a release, can we just direct these changes at master and the pull back into dev?

File: docs/index.rst (hunk 1 above)
Comment (alonamid) [reply docs/index.rst:16]: They aren't completely 100% redundant right now. We should probably make the toolchains installation in initial-setup more similar to the current quick-start form (so peolpe who are just looking for codeline to copy will copy the correct one).
We should also copy the "what's next" part.

File: docs/index.rst [hunk 2]
```diff
@@ -10,10 +10,8 @@ Welcome to Chipyard's documentation!
 
 Chipyard is a framework for designing and evaluating full-system hardware using agile teams.
 It is composed of a collection of tools and libraries designed to provide an integration between open-source and commercial tools for the development of systems-on-chip.
-New to Chipyard? Jump to the :ref:`Chipyard Basics` page for more info.
 
-Quick Start
-===============================
+.. IMPORTANT:: **New to Chipyard?** Jump to the :ref:`Chipyard Basics` page for more info.
```
Comment (alonamid) [inline docs/index.rst:14]: Do we want them to just to "Chipyard Basics", or to "Initial Repository Setup"?

**Comment by colinschmidt** [comment]
Ok @sagark @alonamid I've merged the two quick starts so I think we are ready to go.

File: docs/index.rst [hunk 3]
```diff
@@ -10,89 +10,9 @@ Welcome to Chipyard's documentation!
 
 Chipyard is a framework for designing and evaluating full-system hardware using agile teams.
 It is composed of a collection of tools and libraries designed to provide an integration between open-source and commercial tools for the development of systems-on-chip.
-New to Chipyard? Jump to the :ref:`Chipyard Basics` page for more info.
 
-Quick Start
-===============================
+.. IMPORTANT:: **New to Chipyard?** Jump to the :ref:`Initial Repository Setup` page for more info.
```
Comment (alonamid) [inline docs/index.rst:14]:
```suggestion
.. IMPORTANT:: **New to Chipyard?** Jump to the :ref:`Initial Repository Setup` page for setup instructions.
```

**Comment by alonamid** [comment]
LGTM other than that single suggestions

**Pull request merged**